pkgs/applications/networking/remote/citrix-workspace/generic.nix

```diff
@@ -72,8 +72,6 @@ stdenv.mkDerivation rec {
     gdk-pixbuf
     gnome2.gtkglext
     gnome3.webkitgtk
-    gst-plugins-base
-    gstreamer
```

Comment:
Would not you need `gst_all_1.gstreamer` and the relevant plug-ins?

Reply:
`/nix/store/icmf946gs54m6hzacrydvjfg830abd71-gstreamer-1.16.2/lib` is actually in the RPATH of libraries which need that.